packages/wallets/phantom/package.json

```diff
@@ -30,13 +30,13 @@
         "package": "shx mkdir -p lib/cjs && shx echo '{ \"type\": \"commonjs\" }' > lib/cjs/package.json"
     },
     "peerDependencies": {
-        "@solana/web3.js": "^1.58.0"
+        "@solana/web3.js": "^1.61.0"
```

Comment:
Can you revert these version changes? 1.58.0 is used by all the packages now because it's the minimum version with vTX support.

Reply:
just reverted back

Reply:
> Can you make sure to run the commands to build from source locally? I suspect right now there will be a type error because of this.

You were right. Seems to build successfully now.